examples/pytorch/ogb/directional_GSN/main.py

```diff
@@ -0,0 +1,638 @@
+from ogb.graphproppred import Evaluator
+import torch
+import numpy as np
+from torch.utils.data import Dataset, DataLoader
```

Comment:
Perhaps import `GraphDataLoader` from `dgl.dataloading` instead. While it's simply a wrapper of PyTorch `DataLoader` for this particular example, it has some additional functionalities and we want to enforce a consistent user experience.